### DISCUSSION TITLE
Cron job

### DISCUSSION THREAD

**Discussion by kaisiant** [quirrel-dev/quirrel#1100] [Q&A]
I set to run the cron job every minute. But it only run once not sure why.

```
import { CronJob } from "quirrel/blitz"

export default CronJob(
  "api/test", // 👈 the route that it's reachable on
  ["* * * * *", "Asia/Singapore"], // same as @hourly (see https://crontab.guru/)
  async () => {
    console.log("lol")
  }
)
```
`"dev": "concurrently 'blitz dev' 'quirrel'"`

**Comment by Skn0tt** [comment]
Hi @kaisian97!

There was [a bug](https://github.com/quirrel-dev/quirrel/issues/1089) in a recent version that a fix was released for in [v1.13.1](https://github.com/quirrel-dev/quirrel/releases/tag/v1.13.1). What version are you on, and is the bug fixed by upgrading Quirrel to that version?

If that doesn't fix it, there might be a bug - an issue with detailed reproduction steps (or a repo even!) would be great in that case.
